x/blob/client/testutil/integration_test.go

```diff
@@ -40,17 +39,12 @@ func NewIntegrationTestSuite(cfg cosmosnet.Config) *IntegrationTestSuite {
 	return &IntegrationTestSuite{cfg: cfg}
 }
 
-// Note: the SetupSuite may act flaky especially in CI.
 func (s *IntegrationTestSuite) SetupSuite() {
-	if testing.Short() {
-		s.T().Skip("skipping integration test in short mode.")
-	}
 	s.T().Log("setting up integration test suite")
 
-	net := network.New(s.T(), s.cfg, username)
-
-	s.network = net
-	s.kr = net.Validators[0].ClientCtx.Keyring
+	network := network.New(s.T(), s.cfg, username)
+	s.network = network
```

Comment:
just my 2cents, not blocking, but I don't think we should ever override the name of a package, which was the reason this was called net and not network

Reply:
ohhh great point, that makes a lot of sense. IMO we should probably do that for instances where variables are named `blob` too but will keep in mind for the future and revert this change.

Reply:
4a9d8e50081a3f64ed2544b4b7a1f0df23f2a3f2